examples/helia-browser-verified-fetch/src/App.tsx

```diff
@@ -1,15 +1,19 @@
-import { verifiedFetch } from '@helia/verified-fetch'
+
 import { fileTypeFromBuffer } from '@sgtpooki/file-type'
 import { useCallback, useState } from 'react'
 import { Output } from './Output'
 import { helpText } from './constants'
+import CustomCheckbox from './CustomCheckbox'
+import { getVerifiedFetch } from './utils'
 
-function App (): JSX.Element {
-  const [path, setPath] = useState<string>('')
+function App(): JSX.Element {
+  const [path, setPath] = useState<string>('ipfs://bafybeicklkqcnlvtiscr2hzkubjwnwjinvskffn4xorqeduft3wq7vm5u4')
```

Comment:
do we want the default to be an IPFS string or is just for debugging?

Reply:
Just for debugging. Will remove.